src/lib/features/project/project-service.ts

```diff
@@ -843,6 +843,9 @@ export default class ProjectService {
         );
     }
 
+    /**
+     * @deprecated use `addAccess` instead
+     */
     async addRoleAccess(
```

Comment:
Reason why this isn't deleted right away is because this would break Enterprise. Let's merge the Enterprise PR before we remove this.